test/lib.bash

```diff
@@ -221,8 +221,8 @@ function run_rolling_upgrade_tests {
     --resolvabledomain \
     --https
 
-  # Delete the leftover services.
-  oc delete ksvc --all --cascade='foreground' -n serving-tests
+  # Delete the leftover namespace with services.
+  oc delete namespace serving-tests
```

Comment:
So we keep iterating on this: Why is there services left here in the first place? Is some test not doing what it should?

Reply:
This is how upstream Serving tests work. They start the services in PreUpgrade phase and don't delete them in PostUpgrade phase. Maybe the PostUpgrade phase should delete them in upstream...

Reply:
I'd think it absolutely should, yes.

Reply:
Alright. Can we merge this and then I can work upstream on deleting the services?